workspaces/bazaar/plugins/bazaar/src/components/HomePageBazaarInfoCard/HomePageBazaarInfoCard.tsx

```diff
@@ -93,7 +95,12 @@ export const HomePageBazaarInfoCard = ({
   const [isMember, setIsMember] = useState(false);
 
   const [catalogEntities, fetchCatalogEntities] = useAsyncFn(async () => {
-    const entities = await fetchCatalogItems(catalogApi);
+    let entities = [];
```

Comment:
do you think it could make sense to remove the fetch completely or it makes sense to keep it here for some cases?

Reply:
and yeah same here, was thinking backwards compatibility